.env.sample

```diff
@@ -0,0 +1,3 @@
+AIRTABLE_API_KEY=YOUR_KEY_HERE
+GATSBY_ALGOLIA_APP_ID=9W9HIJ1XKH
+GATSBY_ALGOLIA_SEARCH_KEY=f31eb728872ce9b2be62efd46c3486fb
```

Comment:
So, these are my keys... @bloudermilk what are your keys?

Reply:
@jasonLaster here are the climatescape public keys:
```
GATSBY_ALGOLIA_APP_ID=SKYZAC71CL
GATSBY_ALGOLIA_SEARCH_KEY=5d4b018d3b45f2db1398c038c46cfa8a
```